src/app/modules/main/profile_section/profile/module.nim

```diff
@@ -157,6 +157,9 @@ method storeProfileShowcasePreferences(self: Module,
     revealedAddresses
   )
 
+method setIsFirstShowcaseInteraction(self: Module) =
+  singletonInstance.localAccountSettings.setIsFirstShowcaseInteraction(false)
```

Comment:
The API here looks strange, I'd expect everything in the chain to take the `bool` param and not hardcode the call to `false` here

Reply:
I did it like this bc you can not force the opposite direction in any case, the default is true, but the only change you can do is modifying it to false..

Reply:
OK, fair enough